crates/services/upgradable-executor/src/executor.rs

```diff
@@ -137,7 +137,8 @@ impl<S, R> Executor<S, R> {
         ("0-29-0", 3),
         ("0-30-0", 4),
         ("0-31-0", 5),
-        ("0-32-0", LATEST_STATE_TRANSITION_VERSION),
+        ("0-32-0", 6),
+        ("0-32-1", LATEST_STATE_TRANSITION_VERSION),
```

Comment:
did any changes happen to onchain logic? maybe a new wasm isn't required?

Reply:
I don't think anything has changed, but does it hurt? I guess pushing the wasm up will be work.

Reply:
I feel like it's better to be safe than sorry and just always cut one but 🤷

Reply:
We agreed in the past to do new WASM for each release to avoid any errors. So everything is correct and we need to bump it=)

Plus we upgraded dependencies, so WASM bytecode is different=)